src/test/java/Homework22/LoginPage22.java

```diff
@@ -0,0 +1,36 @@
+package Homework22;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import java.time.Duration;
+
+public class LoginPage22 extends BasePage22 {
+
+    By submitButtonLocator = By.cssSelector("[type='submit']");
+    By emailField= By.cssSelector("[type='email']");
+    By passwordField = By.cssSelector("[type='password']");
```

Comment:
I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

Reply:
We haven't learned about using private but thanks so much for the recommendation. I'll have to look into this!